wormhole-connect/src/views/v2/Redeem/TransactionDetails/index.tsx

```diff
@@ -91,7 +91,7 @@ const TransactionDetails = () => {
             {amount} {sourceTokenConfig.symbol}
           </Typography>
           <Typography color={theme.palette.text.secondary} fontSize={14}>
-            {isFetchingTokenPrices ? (
+            {tokenPrices.isFetching ? (
```

Comment:
I don't think any code actually sets this `isFetching` boolean to true anymore. From what I can tell it's set using `setFetchingPrices` in `stores/tokenPrices.ts` but I don't see that function called anywhere. Maybe we can default `isFetching` to `true` inside the `initialState` value?

Reply:
I decided to call `dispatch(setFetchingPrices())` once in `fetchTokenPrices` instead, wdyt

Reply:
yeah I think once at the beginning makes sense!